app/js/components/orderEntry/addForm/StandardDose.jsx

```diff
@@ -1,170 +1,154 @@
-                </label>
-              </div>
-            </div>
-            <div className="col-sm-2" style={{ paddingLeft: "0", paddingRight: "0" }}>
-              <div className="form-check">
-                <input
-                  className="form-check-input"
-                  type="radio"
-                  name="otherDate"
-                  id="otherDate"
-                  value="option2" />
-                <label className="form-check-label text-right" htmlFor="otherDate">
-                  Other Date
-                </label>
-              </div>
-            </div>
-            <div className="col-sm-2" style={{ paddingLeft: "0" }}>
-              <div className="form-group">
-                <div className="input-group date" id="datetimepicker1">
-                  <input type="text" className="form-control" />
-                  <span className="input-group-addon">
-                    <span className="glyphicon glyphicon-calendar" />
-                  </span>
-                </div>
-              </div>
-            </div>
-          </div>
-
-          <div className="row">
-            <div className="col-sm-11">
-              <div className="form-group">
-                <label htmlFor="notes">Notes:</label>
-                <textarea className="form-control" rows="4" id="notes" />
-              </div>
-            </div>
-          </div>
-        </form>
-
-        <div className="row">
-          <div className="col-sm-3 text-left">
-            <button type="button" className="btn btn-danger">Cancel</button>
-          </div>
-          <div className="col-sm-3 col-sm-offset-5 text-right">
-            <button type="button" className="btn btn-success">Save</button>
-          </div>
+const StandardDose = () => (
+  <div>
+    <form className="simple-form-ui">
+      <div className="grid-row">
+        <div className="column-1">
+          <label name="dose">Dose</label>
+          <input
+            className="number-input"
+            id="dose"
+            name="dose"
+            type="number"
+            min="0"
+            required
+            autoFocus />
+        </div>
+        <div className="column-2">
+          <label name="drugDosingUnits">Units</label>
+          <input
+            id="drugDosingUnits"
+            name="dosingUnit"
+            list="dosingUnits"
+            size="10"
+            required />
+          <datalist id="dosingUnits" >
+            <option value="Kilograms" />
+            <option value="grams" />
+            <option value="milligrams" />
+          </datalist>
+        </div>
+        <div className="column-2">
+          <label name="frequency">Frequency</label>
+          <input
+            id="frequency"
+            name="frequency"
+            list="orderFrequencies"
+            size="10"
+          />
+          <datalist id="orderFrequencies" >
+            <option value="Daily" />
+            <option value="Twice everyday" />
```

Comment:
Did we intentionally hard code these?

Reply:
yes, you advised me to separate the validations from the UI.After this PR is merged, I will make another one with the dynamic data and its validations.